unison-cli/src/Unison/CommandLine/DisplayValues.hs

```diff
@@ -246,10 +246,9 @@ displayPretty pped terms typeOf eval types tm = go tm
       typeOf r >>= \case
         Nothing -> pure $ termName (PPE.suffixifiedPPE pped) r
         Just typ ->
-          pure . P.group $
-            TypePrinter.prettySignaturesCTCollapsed
-              (PPE.suffixifiedPPE pped)
-              [(r, PPE.termName (PPE.suffixifiedPPE pped) r, typ)]
+          pure . P.group
+            . runPretty (PPE.suffixifiedPPE pped)
+            $ TypePrinter.prettySignaturesCTCollapsed [(r, PPE.termName (PPE.suffixifiedPPE pped) r, typ)]
```

Comment:
That's already the case. There's a number of public functions that call runpretty for you, but this particular call site wants some special handling (CTCollapsed, whatever that means)

Reply:
I was suggesting that no public API call site should have to change, including this one. As in, the pretty-printer has a public API (which includes this `prettySignaturesCTCollapsed`) and the type signatures of those functions could be kept identical to what they were before this PR. But maybe since this one is pretty special purpose it doesn't matter so much.

Reply:
I basically don't think we can get away with that for the long term. Ultimately the pretty-printer will be doing monadic stuff to get its own type names and term names, so it'll have to run in some transformer over `IO` or whatever and callers will have to change anyway.